homeassistant/components/backup/manager.py

```diff
@@ -1501,7 +1529,9 @@ def load(self, stored_backups: list[StoredKnownBackup]) -> None:
         self._backups = {
             backup["backup_id"]: KnownBackup(
                 backup_id=backup["backup_id"],
+                failed_addons=[AddonInfo(**a) for a in backup["failed_addons"]],
```

Comment:
It's safer to use explicit parameters when loading persistent data into an object.